merge_state_block/release.json

```diff
@@ -0,0 +1,6 @@
+{
+    "Block": "nio/MergeState",
+    "Version": "1.0.0",
+    "Language": "Python",
+    "Reference": "github.com:nio-blocks/state_change.git@<TODO>"
```

Comment:
The fact that we don't know the commit hash won't be isolated to this run, I think it will happen every time, right? Or else there will always be a subsequent commit where we commit the release.json file.

I think we should take a look at how we do this. First thing that comes to mind is using git tags (or GitHub releases) to make this happen. I've seen other package managers do this.